happypose_ros/happypose_ros/happypose_node.py

```diff
@@ -434,22 +467,31 @@ def _trigger_pipeline(self) -> None:
                 )
             return
 
-        # TODO implement depth info
         rgb_tensor = torch.as_tensor(
             np.stack([cam.get_last_rgb_image() for cam in processed_cameras.values()])
-        )
+        ).permute(0, 3, 1, 2)
+
+        if self._params.use_depth:
+            depth_tensor = torch.as_tensor(
+                np.stack(
+                    [cam.get_last_depth_image() for cam in processed_cameras.values()]
```

Comment:
Can there be aliasing here, i.e. the RGB and depth images do not correspond to the same time?

Reply:
In theory yes. But this is parametrised with synchronised subscriber. I didn't want to assume both have to be synchronised as I really don't trust hardware manufactures and I didn't want to name am assumption that this can only work with realsense